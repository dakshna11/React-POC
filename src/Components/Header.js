import React from 'react';
import {BrowserRouter as Router , Switch, Route, Link , Redirect } from 'react-router-dom';
import HeaderSub from './Header-comp/HeaderSub';
import Home from './Header-comp/Home';
import WashingPoints from './Header-comp/WashingPoints';

let style={ textDecoration: 'none',
            color:'white',
    //":hover:{color:'red'}
}
function Header(props) {
    return (
        
        <div className="header">
            
            <Router>
            <div className="main">
            <div className="H-container">
            <p ><Link  className="words" to={'/'} > Home </Link></p>
            <p><Link className="words" to={'/about'} > About </Link></p>
            <p><Link className="words" to={'/Service'} > Service </Link></p>
            <p><Link className="words" to={'/Price'} > Price </Link></p>
            <p><Link className="words" to={'/Washing Points'} > Washing Points </Link></p>
            <p><Link className="words" to={'/Pages'} > Pages </Link></p>
            <p><Link className="words" to={'/Contact'} > Contact </Link></p>
            </div>
            <div><button id = "btn" onClick>Get Appointment</button></div>
            
            </div>
        
            
        <Switch>
            <Route exact path='/' component={HeaderSub}/>
            <Route path='/about' component={Home}/>
            <Route path='/Washing Points' component={WashingPoints}/>
        </Switch>
        
            
            </Router>
            
        </div>
    );
}

export default Header;