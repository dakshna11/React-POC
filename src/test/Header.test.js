import React from 'react';
import { shallow } from 'enzyme';
import Header from "../Components/Header";


let wrapped = shallow(<Header></Header>);
describe('Header', () => {
  it('should render the Header Component correctly', () => {   
    expect(wrapped).toMatchSnapshot();
  });
  
})
