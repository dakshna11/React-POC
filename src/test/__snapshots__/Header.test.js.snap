// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the Header Component correctly 1`] = `
<div>
  <Sticky
    className="sticky"
    stickyClassName="sticky"
    stickyStyle={Object {}}
    wrapperClassName=""
  >
    <div
      className="header"
    >
      <div
        className="main"
      >
        <div
          className="H-container"
        >
          <p>
            <Link
              className="words"
              to="/"
            >
               Home 
            </Link>
          </p>
          <p>
            <Link
              className="words"
              to="/about"
            >
               About 
            </Link>
          </p>
          <p>
            <Link
              className="words"
              to="/Service"
            >
               Service 
            </Link>
          </p>
          <p>
            <Link
              className="words"
              to="/Price"
            >
               Price 
            </Link>
          </p>
          <p>
            <Link
              className="words"
              to="/Washing Points"
            >
               Washing Points 
            </Link>
          </p>
          <p
            className="dropdown"
          >
            <Link
              className="words"
              to="/Pages"
            >
              Pages
              <i
                className="fa fa-caret-down"
                id="caret-down"
              />
            </Link>
            <div
              className="dropdown-content"
            >
              <p>
                Blog Grid
              </p>
              <p>
                Detail Page
              </p>
              <p>
                Team Member
              </p>
              <p>
                Schedule Booking
              </p>
            </div>
          </p>
          <p>
            <Link
              className="words"
              to="/Contact"
            >
               Contact 
            </Link>
          </p>
        </div>
        <div>
          <button
            id="btn"
          >
            Get Appointment
          </button>
        </div>
      </div>
    </div>
  </Sticky>
</div>
`;
